content/config.yml

```diff
@@ -18,12 +18,15 @@ nav:
         url: '/blog/'
   - text: 'services'
     subnav:
-      - text: 'design-development'
+      - text: 'planning'
+        display_text: 'product strategy & planning'
+        url: '/services/planning/'
+      - text: 'development'
         display_text: 'application design & development'
-        url: '/work/services/'
-      - text: 'design-systems-training'
-        display_text: 'design systems'
-        url: '/services/design-systems-training/'
+        url: '/services/'
+      - text: 'design-systems'
+        display_text: 'integrated design systems'
+        url: '/services/design-systems/'
```

Comment:
Hrm, it's a bummer to change existing URLs...

Reply:
Yeah, I usually tried to avoid it, but hoped this page was new enough…? Would you prefer leaving it how it was?

Reply:
Yeah, this one maybe is. But the `/work/services/` has definitely been around longer, and that's also changed here.

And if we _are_ changing them, I'm not clear why `/speaking/` still gets a top-level URL.